src/components/v5/common/ActionSidebar/hooks/useActionDescriptionMetadata.tsx

```diff
@@ -59,6 +66,22 @@ export const useActionDescriptionMetadata = () => {
   const { user } = useAppContext();
   const { colony } = useColonyContext();
 
+  const { data: colonyContributorData } = useGetColonyContributorQuery({
+    variables: {
+      id: getColonyContributorId(
+        colony?.colonyAddress || '',
+        user?.walletAddress || '',
+      ),
+      colonyAddress: colony?.colonyAddress || '',
+    },
+  });
+
+  const contributor = colonyContributorData?.getColonyContributor;
+  const { isVerified } = contributor || {};
+  const domains = useContributorBreakdown(contributor);
+  const userStatus = (contributor?.type?.toLowerCase() ??
+    null) as ContributorTypeFilter | null;
+
```

Comment:
Does this query need to be done here? Can it not happen in the `UserPopover` itself?